### DISCUSSION TITLE
Disable warnings

### DISCUSSION THREAD

**Discussion by Tsvetina17** [community/community#139162] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi I am relatively new to Github. I noticed that I get a lot of warnings under some of my graphs in Jupyter when I upload them to GitHub. I tried using the warnings module from my python but it doesn't help. I looked for advice on Stack Overflow but nothing I tried works. Could someone please help me?

![image](https://github.com/user-attachments/assets/903e8250-4b31-4241-aa68-d432d0d80bec)

**Comment by davevad93** [comment, marked as answer]
Hi @Tsvetina17 ,

To remove all warnings in my notebooks, I always do the following:

```python
# Import the warnings module
import warnings

# Ignore all warnings
warnings.filterwarnings("ignore", category=FutureWarning)
```

**Reply by Tsvetina17** [reply]
thank you, you really made my day :)

**Reply by davevad93** [reply]
You're welcome @Tsvetina17, happy to help! 👍
